prettier.config.js

```diff
@@ -1,2 +1,3 @@
-// take default
-module.exports = {};
+module.exports = {
```

Comment:
This change isn't needed, since 3.0.0, the default for trailing comma is `all`. See https://prettier.io/docs/en/options#trailing-commas

Reply:
OK, seems it didn't work well in my computer, but I guess it's some of my own problem. I'll remove this, and find out.